solutions/cses-2431.mdx

```diff
@@ -0,0 +1,219 @@
+---
+id: cses-2431
+source: CSES
+title: Digit Queries
+author: Nathan Gong
+---
+
+## Explanation
+
+_Note that this is very tricky for a bronze problem!_
```

Comment:
would this be better in a warning maybe? I'm not sure what _ does though

Reply:
makes it italicized